test/sass/_breakpoint.scss

```diff
@@ -24,10 +24,10 @@
 
   @include test('Breakpoint (Only Range) [function]') {
     $test: breakpoint(medium only);
-    $expect: '(min-width: 40em) and (max-width: 63.9375em)';
+    $expect: '(min-width: 40em) and (max-width: 63.9999em)';
```

Comment:
There is some tests to do in IE/Edge to ensure this works properly (or at least better than now). See #10808

Reply:
Looking at #10808, wouldn't `.9375` be just as much a problem as `.9999`? I don't see this as introducing new issues. Thoughts?

Reply:
True but I prefer testing. Also, is there any reason for `.9999` and not `.999999` ?

Reply:
This is from my [previous comment](https://github.com/zurb/foundation-sites/pull/10978#discussion_r170259770), but since all the em math works in 16ths, values end up at 4 decimals. Do you have a suggestion other than `.0001` which would result in `.9999`?

Reply:
According to the W3C CSS3 reference:

> CSS theoretically supports infinite precision and infinite ranges for all value types; however in reality implementations have finite capacity. UAs should support reasonably useful ranges and precisions.
> -- https://www.w3.org/TR/css-values-3/#numeric-types

It is common is CSS to write values with 6 decimals as it is precise enough to ensure a sub-pixel difference. But when dealing with breakpoints I'm not sure it's what we want.

Reply:
I understand your concern. What can I do to move this along?

Reply:
@ncoden is there a testing procedure you'd like me to follow here?

Reply:
@JasonMiller I just have some tests to do across browsers, when I have the time for.